nessie/src/main/java/org/apache/iceberg/nessie/NessieCatalog.java

```diff
@@ -202,7 +202,16 @@ protected TableOperations newTableOps(TableIdentifier tableIdentifier) {
   protected String defaultWarehouseLocation(TableIdentifier table) {
     String location;
     if (table.hasNamespace()) {
-      location = SLASH.join(warehouseLocation, table.namespace().toString(), table.name());
+      String baseLocation;
+      try {
+        baseLocation =
+            loadNamespaceMetadata(table.namespace())
+                .getOrDefault(
+                    "location", SLASH.join(warehouseLocation, table.namespace().toString()));
```

Comment:
Thanks for the PR.

How does the user know he needs to configure the namespace property as "location"? hive calls it a "database.locationUri" I guess. 

I think we need to standardize it for all the catalogs or document the supported namespace properties per catalog.

Reply:
> If the namespace it's created using flink, spark or hive at least like 'CREATE namespace t LOCATION 'somewhere'' it will populate this "location" property.

Thanks. Then it is already standardized. I was not aware of this (I assumed the user has to manually configure the property bag). I agree with your code changes.